examples/karaoke.rs

```diff
@@ -92,6 +125,16 @@ fn main() -> Result<(), Error> {
             // Play back the processed audio capture.
             out_buffer.copy_from_slice(&processed);
             processor.process_render_frame(out_buffer).unwrap();
+            // Handle mono to mono/stereo conversion
+            if output_channels == 1 {
+                out_buffer.copy_from_slice(&processed);
+            } else {
```

Comment:
Might be good to leave a note that we're assuming if it's not 1 output channel, it's 2. As far as I can tell it could be an arbitrary number, but this is also example code so not a big deal.